packages/ui/pages/index.tsx

```diff
@@ -55,7 +48,7 @@ const ReportQuery = gql`
 `;
 
 interface SuggestionOverviewProps {
-  topSuggestions: NexusGenFieldTypes['TopSuggestions'][];
+  topSuggestions: IndexReportQuery['report']['topSuggestions'];
```

Comment:
Two Options:

1. Pull this out into a type and export it for usage via `export type TopSuggestion = IndexReportQuery['report']['topSuggestions']`
2. Use a graphql fragment and then export it for usage in multiple places

Reply:
updated, good suggestion.